ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ZAAppDelegateConnectorChromecastProtocol.h
+//  ZappAppConnector
+//
+//  Created by Avi Levin on 21/12/2017.
+//  Copyright © 2017 Applicaster Ltd. All rights reserved.
+//
+@protocol ZAAppDelegateConnectorChromecastProtocol
```

Comment:
Why isn't this in Swift?

Reply:
Because the connector code is already in OBJ C.

Reply:
And what is the problem of using Swift for this protocol even if the class that is going to implement it is objective c?